modules/appeals_api/app/swagger/legacy_appeals/v0/swagger_dev.json

```diff
@@ -46,33 +46,17 @@
         "description": "Returns eligible legacy appeals for a Veteran. A legacy appeal is eligible if a statement of the case (SOC)  or supplemental statement of the case (SSOC) has been declared, and if the date of declaration is within the last 60 days.",
         "parameters": [
           {
-            "in": "header",
-            "name": "X-VA-SSN",
-            "required": false,
-            "description": "Either X-VA-SSN or X-VA-File-Number is required. Example X-VA-SSN: 123456789",
-            "example": "706547821",
-            "schema": {
-              "$ref": "#/components/schemas/X-VA-SSN"
-            }
-          },
-          {
-            "in": "header",
-            "name": "X-VA-File-Number",
-            "required": false,
-            "description": "Either X-VA-SSN or X-VA-File-Number is required. Example X-VA-File-Number: 123456789",
-            "schema": {
-              "$ref": "#/components/schemas/X-VA-File-Number"
-            }
-          },
-          {
-            "in": "header",
-            "name": "X-VA-ICN",
+            "name": "icn",
+            "description": "Veteran's Master Person Index (MPI) Integration Control Number (ICN)",
+            "example": "1234567890V123456",
```

Comment:
I have not yet updated the global default ICN in icn.json to match the newly updated test user account (I plan to do that in a future PR for API-31676)